pages/Configuring/Binds.md

```diff
@@ -137,14 +137,31 @@ You can view your switches in `hyprctl devices`.
 You can trigger multiple actions with one keybind by assigning multiple binds to
 one combination, e.g.:
 
-```
+```ini
 # to switch between windows in a floating workspace
 bind = SUPER,Tab,cyclenext,          # change focus to another window
 bind = SUPER,Tab,bringactivetotop,   # bring it to the top
 ```
 
 The keybinds will be executed in the order they were created. (top to bottom)
 
+### Description
+
+You can describe you keybind with the description flag.
```

Comment:
your

Reply:
fixed in commit c6cd521d8e848bd3ae5978ab118696eca5ff7e22